bitrise.yml

```diff
@@ -1449,7 +1505,7 @@ trigger_map:
 - push_branch: release/v112
   workflow: SPM_Deploy_Prod_Beta
 - pull_request_target_branch: main
-  pipeline: pipeline_multiple_shards
+  workflow: perfherder-test
```

Comment:
@jjSDET: Now that we have seen that the workflow works, we need to set the default pipeline back here before merging.